google/cloud/firestore_v1/document.py

```diff
@@ -255,7 +255,7 @@ def update(self, field_updates, option=None) -> Any:
         write_results = batch.commit()
         return _first_write_result(write_results)
 
-    def delete(self, option=None) -> Any:
+    def delete(self, option=None) -> Coroutine:
```

Comment:
This method returns a single `google.protobuf.timestamp_pb2.Timestamp` instance.
```suggestion
    def set(self, option=None) -> timestamp_pb2.WriteResult:
```